login-workflow/example/src/screens/CustomPage.tsx

```diff
@@ -0,0 +1,154 @@
+                                <Select
+                                    value={app.language}
+                                    onChange={changeAppLanguage}
+                                    variant={'standard'}
+                                    sx={{
+                                        padding: '4px',
+                                        minWidth: '160px',
+                                        margin: '16px',
+                                        backgroundColor: 'transparent',
+                                        color: Colors.white[50],
+                                        '& .MuiSelect-icon': {
+                                            color: Colors.white[50],
+                                        },
+                                    }}
+                                >
+                                    <MenuItem value={'en'}>English</MenuItem>
+                                    <MenuItem value={'es'}>Spanish</MenuItem>
+                                    <MenuItem value={'fr'}>French</MenuItem>
+                                    <MenuItem value={'zh'}>Chinese</MenuItem>
+                                    <MenuItem value={'pt'}>Portugese</MenuItem>
+                                </Select>
+                            </FormControl>
+                        </Box>
+                        {/* <UserMenu
+                            onClick={(): void => setOpen(!open)}
+                            avatar={<Avatar>AV</Avatar>}
+                            menuGroups={[
+                                {
+                                    items: [
+                                        {
+                                            icon: <AccountBox />,
+                                            title: `${t('USER_MENU.MY_ACCOUNT')}`,
+                                            onClick: (): void => {
+                                                setOpen(false);
+                                            },
+                                        },
+                                        {
+                                            icon: <ExitToApp />,
+                                            title: `${t('USER_MENU.LOG_OUT')}`,
+                                            onClick: (): void => {
+                                                setOpen(false);
+                                                logOut();
+                                            },
+                                        },
+                                    ],
+                                },
+                            ]}
+                        /> */}
+                    </Toolbar>
+                </AppBar>
+            </Box>
+            <Box sx={containerStyles}>
+                <EmptyState
+                    sx={emptyStateContainerStyles}
+                    icon={<Event fontSize={'inherit'} />}
+                    title={`${t('TOOLBAR_MENU.HOME_PAGE')}`}
```

Comment:
Update the text here, it's the same as homepage

Reply:
This is just for testing purposes so the reviewer can test (forgot to mention). Will delete this .

Reply:
> This is just for testing purposes so the reviewer can test (forgot to mention). Will delete this .

@joebochill

Reply:
Cool, I tested the redirects via deep linking (for custompage) and they look good.